src/jwt/jwtparser.c

```diff
@@ -59,13 +59,19 @@ static cjose_jwk_t *get_jwk(JWS *jws)
     if (!doc)
         goto errorExit;
 
-    keyid = DIDURL_FromString(JWS_GetKeyId(jws), NULL);
-    if (!keyid)
-        goto errorExit;
-
-    key = DIDDocument_GetPublicKey(doc, keyid);
-    if (!key)
-        goto errorExit;
+    if (!JWS_GetKeyId(jws)) {
+        keyid = DIDDocument_GetDefaultPublicKey(doc);
+        key = DIDDocument_GetPublicKey(doc, keyid);
```

Comment:
Same with "else" branch to destroy "keyid" ?

Reply:
```c
DIDDocument_GetDefaultPublicKey
```
The return value from this api is the point from 'doc', not create the new one, so no need to free the return value, which is different with 'DIDURL_NewFromDid'.

Reply:
> Same with "else" branch to destroy "keyid" ?

The return value from this api is the point from 'doc', not create the new one, so no need to free the return value, which is different with 'DIDURL_NewFromDid'.